code/modules/mapping/mapping_helpers.dm

```diff
@@ -205,3 +205,22 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 		CRASH("Wrong disease type passed in.")
 	var/datum/disease/D = new disease_type()
 	return list(component_type,D)
+
+/obj/effect/mapping_helpers/dead_body_placer
+	name = "Dead Body placer"
+	icon_state = "deadbodyplacer"
+
+/obj/effect/mapping_helpers/dead_body_placer/Initialize(mapload)
+	var/area/a = get_area(src)
+	var/trays = list()
+	for (var/i in a.contents)
+		if (istype(i, /obj/structure/bodycontainer/morgue))
+			trays += i
+	for (var/i = 1 to 2)
+		var/obj/structure/bodycontainer/morgue/j = pick(trays)
+		var/mob/living/carbon/human/h = new /mob/living/carbon/human(j.loc, 1)
```

Comment:
the morgue's `loc` is the turf so don't you want it inside the morgue instead?

Reply:
All addressed.